pkg/promlib/models/query.go

```diff
@@ -119,22 +120,15 @@ var safeResolution = 11000
 
 // QueryModel includes both the common and specific values
 type QueryModel struct {
-	PrometheusQueryProperties `json:",inline"`
-	CommonQueryProperties     `json:",inline"`
+	PrometheusQueryProperties    `json:",inline"`
+	sdkapi.CommonQueryProperties `json:",inline"`
 
 	// The following properties may be part of the request payload, however they are not saved in panel JSON
 	// Timezone offset to align start & end time on backend
 	UtcOffsetSec int64  `json:"utcOffsetSec,omitempty"`
 	Interval     string `json:"interval,omitempty"`
 }
 
-// CommonQueryProperties is properties applied to all queries
-// NOTE: this will soon be replaced with a struct from the SDK
-type CommonQueryProperties struct {
-	RefId      string `json:"refId,omitempty"`
-	IntervalMs int64  `json:"intervalMs,omitempty"`
```

Comment:
Looks like prometheus has always read this as an int64 although upstream it could be float64 -- in practice I think it is always an int so I'm not sure the best approach to resolve